### PR TITLE
Implemented comm 

### PR DESCRIPTION
**Pull request by ShisuiMadara**
Complete implementation with all three flags 
- 1
- 2
- 3
 following the POSIX output standard. The program was tested locally and the output was a success.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -54,4 +54,6 @@ members = [
     "who",
     "whoami",
     "yes",
+    "comm",
+    "join",
```
Comment (GrayJack) [inline Cargo.toml:58]: Are this PR related to `join` as well?

File: comm/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[package]
+name = "comm" # Edit this
+version = "0.1.0"
+authors = ["shisuimadara adyityasingh@gmail.com"] # Edit this
```
Comment (GrayJack) [inline comm/Cargo.toml]: The format we used so far was `name <email>`

File: comm/src/cli.rs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        .arg(
+            Arg::new("2")
+                .short('2')
+                .help("Suppress lines unique to file2.")
+        )
+        .arg(
+            Arg::new("3")
+                .short('3')
+                .help("Suppress lines common to both files.")
+        )
```
Comment (GrayJack) [inline comm/src/cli.rs:25]: So far we have been adding short and long on all flags. Since those don't have a long version on any other implementation, you are free to create one

File: comm/src/main.rs [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+        Err(e) => {
+            match e.kind() {
+                ErrorKind::NotFound => {
+                    eprintln!("comm: {}: No such file or directory", file1)
+                },
+                ErrorKind::PermissionDenied => {
+                    eprintln!("comm: {}: Permission denied", file1)
+                },
+                _ => eprintln!("comm: {}: Unknown error", file1),
+            }
```
Comment (GrayJack) [inline comm/src/main.rs]: There was any particular reason behind not printing the error message of the `io::Error`?

File: comm/src/main.rs [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+        Err(e) => {
+            match e.kind() {
+                ErrorKind::NotFound => {
+                    eprintln!("comm: {}: No such file or directory", file2)
+                },
+                ErrorKind::PermissionDenied => {
+                    eprintln!("comm: {}: Permission denied", file2)
+                },
+                _ => eprintln!("comm: {}: Unknown error", file2),
+            }
```
Comment (GrayJack) [inline comm/src/main.rs]: There was any particular reason behind not printing the error message of the io::Error?

**Review by GrayJack — CHANGES_REQUESTED** [review]
The code looks very good for the most part, just have a few questions to clarify and some nitpicks.

I also believe that you committed and pushed some code not related to the PR, files about the `join` util

File: Cargo.toml (hunk 1 above)
Comment (ShisuiMadara) [reply Cargo.toml:58]: I started working on join and then made an initial commit, the changes were reflected here too.

File: comm/Cargo.toml (hunk 2 above)
Comment (ShisuiMadara) [reply comm/Cargo.toml]: Ok, got it, updated.

File: comm/src/main.rs (hunk 4 above)
Comment (ShisuiMadara) [reply comm/src/main.rs]: I am sorry I left ambiguity there. Fixed it.

File: comm/src/main.rs (hunk 5 above)
Comment (ShisuiMadara) [reply comm/src/main.rs]: Sorry, fixed it.

**Comment by ShisuiMadara** [comment]
I made all the changes, @GrayJack kindly consider merging.
